client/lib/credit-card-details/validation.js

```diff
@@ -177,6 +233,20 @@ function getErrors( field, value, cardDetails ) {
 	);
 }
 
+/**
+ *
+ * @param {object} cardDetails - a map of credit card field key value pairs
+ * @returns {object|null} If match is found,
+ * an object containing rule sets for specific credit card processing providers,
+ * otherwise `null`
+ */
+function getAdditionalFieldRules( { country } ) {
+	if ( isEbanx( country ) ) {
```

Comment:
I think we need this to be on a provider+country level.

We'll probably use ebanx on a number of countries, but not all of those countries have the same fields. 
So instead of ` return ebanxFieldRules()`, we'd want `return ebanxFieldRules( country )`

Reply:
Good point. Thanks for the suggestion.